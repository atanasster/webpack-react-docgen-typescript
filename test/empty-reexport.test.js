import compiler from './compiler.js';

jest.setTimeout(100000);

test('empty index', async () => {
  const stats = await compiler('./examples/reexport.ts');
  const jsonStats = stats.toJson();
  const output = jsonStats.modules[5].source;
  expect(output).toMatchSnapshot();
});
