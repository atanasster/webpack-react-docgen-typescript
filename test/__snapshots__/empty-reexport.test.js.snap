// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`empty index 1`] = `undefined`;
